inference-engine/tests/functional/plugin/shared/include/single_layer_tests/gather_elements.hpp

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "shared_test_classes/single_layer/gather_elements.hpp"
+
+namespace LayerTestsDefinitions {
+
```

Comment:
Why is it empty? I assume here should be test definition like `TEST_P(GatherElementsLayerTest, CompareWithRefs)`

Reply:
That test definition is already in `inference-engine/tests/functional/shared_test_classes/src/single_layer/gather_elements.cpp` file. 

So I didn't add that code to avoid dupicate declarations.

Reply:
Changed the location of the test definition from `inference-engine/tests/functional/shared_test_classes/src/single_layer/gather_elements.cpp` to `inference-engine/tests/functional/plugin/shared/include/single_layer_tests/gather_elements.hpp`.

And, Fixed header in `inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/gather_elements.cpp` file.